pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc

```diff
@@ -1,21 +1,22 @@
 [id="previewing-modules-in-pantheon_{context}"]
-= Previewing modules in {ProductShortName}
+= Previewing {ContentTerm} in {ProductShortName}
 
-Preview a module that has been imported in {ProductShortName}. 
+[role="_abstract"]
+Preview {ContentTerm} that has been imported in {ProductShortName}. 
```

Comment:
...that has been imported *into*...
(rather than *in*)

Reply:
done